Documentation/project-docs/using-nightly-builds.md

```diff
@@ -0,0 +1,244 @@
+# Using nightly builds of .NET Core 2.0
+
+This document provides the steps necessary to consume a nightly build of .NET Core 2.0 runtime and SDK.
+
+Please note that these steps are likely to change as we're simplifying this experience. Make sure to consult this document often.
+
+## Install prerequisites
+
+1. Acquire the latest nightly .NET Core SDK 2.0
+
+- [Win 64-bit Latest Zip](https://dotnetcli.azureedge.net/dotnet/Sdk/master/dotnet-dev-win-x64.latest.zip) [Installer](https://dotnetcli.azureedge.net/dotnet/Sdk/master/dotnet-dev-win-x64.latest.exe)
+- [macOS 64-bit Latest Tar](https://dotnetcli.azureedge.net/dotnet/Sdk/master/dotnet-dev-osx-x64.latest.tar.gz) [Installer](https://dotnetcli.azureedge.net/dotnet/Sdk/master/dotnet-dev-osx-x64.latest.pkg)
+- [Others](https://github.com/dotnet/cli/blob/master/README.md#installers-and-binaries)
+
+To setup the SDK, download the zip, extract it somewhere, and add the root folder to your path. Alternatively, always fully qualify the path to `dotnet` in the root of this folder for all instructions in this document.
+
+Note: The installer will put `dotnet` globally in your path, which you might not want for dogfooding daily toolsets.
+
+After setting up `dotnet` you can verify you are using the newer version  with `dotnet --info`. The version should be greater than 2.0.0-*.
+
+## Setup the project
+
+1. Create a new project
+    - Create a new folder for your app
+    - `cd` into it and run `dotnet new console`
+
+2. Restore packages so that you're ready to play:
+
+```
+$ dotnet restore
+```
```

Comment:
Is manually running `dotnet restore` still necessary, since .Net Core SDK 2.0 does it automatically?

I think this also applies to other mentions of `dotnet restore` below.